doc/README_macos_clanggfortran.txt

```diff
@@ -101,11 +103,10 @@ the following commands should suffice to build the model.
 
 export INSTALL_PREFIX=/usr/local/NCEPLIBS-ufs-v2.0.0
 
-export CC=clang-9
-export FC=gfortran-9
-export CXX=clang++-9
-export PATH="/usr/local/opt/llvm/bin:$PATH"
-export LD_LIBRARY_PATH="/usr/local/opt/llvm/lib:$LD_LIBRARY_PATH"
+export PATH="/usr/local/opt/llvm@10/bin:$PATH"
```

Comment:
What is the point of these lines?

Reply:
But I approve

Reply:
You need those lines to find the correct version of clang (the homebrew-installed LLVM clang with OpenMP support) and to link to the OpenMP library. This is also what homebrew tells you to do. Without them, cmake can't find the OpenMP C library, no matter what you do.